vs-code/vscode.md

```diff
@@ -12,7 +12,11 @@ About 1 hour
 
 ### Motivation
 
-Most software is written using text editors. Visual Studio Code (VS Code) is a very popular open-source text editor developed and maintained by Microsoft. Having a good understanding of how to use VS Code will enable you to code and use other text editors with ease.
+Most software is written using text editors. Visual Studio Code (VS Code) is a very popular open-source text editor developed and maintained by Microsoft. Having a good understanding of how to use VS Code will enable you to code and use other text editors with ease. Most of the developers prefer using VS Code for writing codes because of its flexiblity and user interface. It provides almost all the facilities that makes the taks easier.
+
+At its heart, Visual Studio Code features a lightning fast source code editor, perfect for day-to-day use. With support for hundreds of languages, VS Code helps you be instantly productive with syntax highlighting, bracket-matching, auto-indentation, box-selection, snippets, and more.
```

Comment:
Please remove this block- it’s just a wordy product pitch.

Instead, tell them what kind of developers prefer VSCode, or a famous company or coder that is aided by VSCode.

Reply:
@alodahl couldn't find specific examples of coders who use vs code. most of the web developers and front end develpoer uses this.

Reply:
Most of them?  Where did you get this data?

Reply:
Or do you mean that most users of VSCode are web developers and front end developers?

Reply:
yes most of the users are web developers or front-end developers